### ISSUE TITLE
Tag info not cleared when changing track

### ISSUE THREAD

**Issue by GlodNet** [notandy/ympd#85]
If you change from a track *with* tags to one *without*, YMPD doesn't clear the old info.  So, for example, the old artist and album info is left on-screen when it should've been blanked out.

Regards,

Mark.

**Comment by GlodNet** [comment]
I forgot to list version info (which is invoked via "-v" and *not* "-V" as the text says):

ympd  1.2.3
Copyright (C) 2014 Andrew Karpow <andy@ndyk.de>
built Oct 19 2014 22:02:18 (4.6.3)

**Comment by GlodNet** [comment]
Thanks for the new version.

Unfortunately, while the radio streams work, the blanking of non-existant tags doesn't.  It behaves as the old version did; viz, if I refresh the page (via pressing F5) then the missing tags are erased but otherwise they remain on-screen.  (I'm accessing the web page via FireFox 40.0 running on Mint 17.2.)

Note that this is the first time that I've built a program from source on the RPi, so there's a possibility that I've done something wrong.  (There was a brief bit of faffing until I realised that LIBMPDCLIENT2 *and* LIBMPDCLIENT-DEV are required...)

Hope this helps.

Regards,

Mark.
